EarlyBinding/src/EXPRESS/EXPRESSReferenceImpl.h

```diff
@@ -16,6 +16,10 @@
 	along with this program. If not, see <http://www.gnu.org/licenses/>.
 */
 
+#pragma once
+#ifndef OpenInfraPlatform_EarlyBinding_EXPRESSOptional_76c70f6e_bab2_48da_b2bc_9434b6adf3dc_i
+#define OpenInfraPlatform_EarlyBinding_EXPRESSOptional_76c70f6e_bab2_48da_b2bc_9434b6adf3dc_i
+
```

Comment:
Is this again the same include guard as in EXPRESSOptional?

Reply:
resolved in 7a9a70c